packages/pigeon/lib/cpp_generator.dart

```diff
@@ -261,7 +272,7 @@ void _writeDataClassImplementation(Indent indent, Class klass, Root root) {
       root.enums.map((Enum x) => x.name).toSet();
 
   indent.addln('');
-  indent.writeln('/* ${klass.name} */');
+  indent.writeln('// ${klass.name}');
```

Comment:
If you also rename the constant to `commentPrefix` you could use that everywhere you're currently using `//` directly.